newlib/libm/test/meson.build

```diff
@@ -157,27 +164,34 @@ foreach target : targets
   _c_args = double_printf_compile_args + _c_args
   _link_args = double_printf_link_args + _link_args
 
-  test(test_name,
-       executable(test_name, math_test_src,
-		  c_args: _c_args,
-		  objects: objs,
-		  link_with: libs,
-		  link_args: _link_args,
-		  link_depends: _link_depends,
-		  include_directories: inc),
-       depends: bios_bin,
-       env: test_env)
+  foreach i : range(max_test_part)
+    test_part = i.to_string()
```

Comment:
A few of the run scripts rely on the `target` piece being at the end of the executable name. That means pulling the computation of `test_name` inside this loop and placing the `test_part` after `'math_test_'` and before `target`.

Reply:
Fixed.